### PR TITLE
Layer GEOMTRANSFORM is in ground units, not in pixels.

### PR DESCRIPTION
**Pull request by ochriste**

### CODE REVIEW

File: en/mapfile/layer.txt [hunk 1]
```diff
@@ -358,7 +358,7 @@ GEOMTRANSFORM [<expression>|<Javascript file>]
          triple: LAYER; GEOMTRANSFORM; buffer
 
       - *(buffer([shape],dist))*: Buffer the geometry (`[shape]`)
-        using `dist` pixels as buffer distance. For polygons, a
+        using `dist` ground units as buffer distance. For polygons, a
```
Comment (yjacolin) [inline en/mapfile/layer.txt]: ground unit or projection units?

Comment (ochriste) [reply en/mapfile/layer.txt]: see https://github.com/ochriste/docs/blob/b35e9490ccd3a02eca6b476285d3fbda32243473/en/mapfile/layer.txt#L396-L402

Comment (yjacolin) [reply en/mapfile/layer.txt]: thanks it looks like it is the same, just another way to name it.

**Comment by havatv** [comment]
@ochriste, many thanks for helping us improve the documentation!
This applies to the current version of Mapserver, so I suggest that it is applied to branch-7-0.  If you think it is too much trouble to do it for that branch, I can do it for you.  Please let me know what you decide.

**Comment by ochriste** [comment]
@havatv done

**Comment by havatv** [comment]
Done for branch-7-0, so closing.

**Pull request closed without merging**